api/_mixins/LinkStyle__HTMLLinkElement.json

```
@@ -31,7 +31,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "3"
```

```suggestion
              "version_added": "1"
```

https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_html.cpp?rev=4#L314